### ISSUE TITLE
input position breaks after tab click

### ISSUE THREAD

**Issue by utd1878** [angular/material2#9853]
#### Bug, feature request, or proposal:
When placing input control in mat-tab and then moving from tab to tab, the input position is above where it shows.

#### What is the expected behavior?
That the input position would be where the element is on the dom
![image](https://user-images.githubusercontent.com/5215011/36003607-12878454-0cec-11e8-97a2-fab836301c85.png)

#### What is the current behavior?
In put position is on top of where it should be.
![image](https://user-images.githubusercontent.com/5215011/36003631-1dad4ba2-0cec-11e8-9694-9b10acd9adc6.png)

#### What are the steps to reproduce?
Providing a StackBlitz reproduction is the *best* way to share your issue. <br/>
StackBlitz starter: https://goo.gl/wwnhMV<br/>

I'm having trouble reproducing this on a plunkr, so I'm not sure what else to share or how I can share.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
angular 4.4.6, Material 5.1.1

#### Is there anything else we should know?
I'm using this in an Electron app

**Comment by utd1878** [comment]
I've disabled the animation on the tab as explained in this thread, and the functionality behaves as expected...
https://github.com/angular/material2/issues/2706

**Comment by pklaschka** [comment]
The same thing happens to me (only "worse"), as the whole layout breaks with animations enabled. 

If I replace the ```BrowserAnimationsModule``` with the ```NoopAnimationsModule``` in my ```app.module.ts```, everything works as expected. But with animations enabled the whole layout (not just inputs) breaks after the first time I move to a different tab.

For me, it's also an Electron app I develop and I additionally use ```@angular/flex-layout```.

### Dependencies in my case:
```
[...]

"@angular/animations": "^5.2.1",
"@angular/cdk": "^5.2.1",
"@angular/flex-layout": "^2.0.0-beta.12",
"@angular/material": "^5.2.1",
"hammerjs": "^2.0.8"

[...]
```
With all the Angular-Specific dependencies (```@angular/[…]```) at version ```^5.2.0```.

### Screenshots
What it looks like with the ```NoopAnimationsModule```:
![bildschirmfoto 2018-02-15 um 19 59 12](https://user-images.githubusercontent.com/25147704/36275388-fc9bc6b0-128a-11e8-8d25-b542a9161231.png)

What it looks like with the ```BrowserAnimationsModule```:
![bildschirmfoto 2018-02-15 um 20 00 32](https://user-images.githubusercontent.com/25147704/36275403-0c8a814c-128b-11e8-9f72-5e8206a28929.png)

**Comment by mmalerba** [comment]
This appears to be exclusive to electron apps, it works fine in a stackblitz: https://stackblitz.com/edit/angular-gse18b?file=app/tabs-overview-example.html

**Comment by pklaschka** [comment]
For me, first setting the Angular-Dependencies from version ```^5.2.1``` to ```5.2.1``` and after that setting them to ```^5.2.5``` fixed it (I have no idea why, though).

I don't think it is (or in my case was) an Electron-related issue, though, as I didn't change anything about Electron-related dependencies (I only played around with the Angular version numbers; all my Electron dependencies have "locked" version numbers).

**Comment by mmalerba** [comment]
In that case it sounds like it was probably an issue with Angular animations that was fixed sometime between 5.2.0 and 5.2.5.
